app/models.py

```diff
@@ -131,7 +131,7 @@ def __str__(self):
 
 
 class Request(models.Model):
-    skill = models.ForeignKey(Skill, on_delete=models.CASCADE),
+    skill = models.ForeignKey(Skill, on_delete=models.CASCADE, null=True)
```

Comment:
Why are we making skill nullable?

Reply:
@Del-sama I think i encountered a block then and prolly @kosyfrances or @Nkoli asked to null the skill cant recall who and the actual reason.

Reply:
I think this was because you updated something in the model and it said we should add a default one off when you wanted to run migrations